src/index.html

```diff
@@ -9,6 +9,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo__link">
+        <img
+          src="images/logo.png"
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="#samsung">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#smartphones">Smartphones</a>
          </li>
```

Add blank line between multiple sibling blocks of html